### DISCUSSION TITLE
iptv goes down, help only a restart

### DISCUSSION THREAD

**Discussion by jensus11** [fabianishere/udm-iptv#183] [Q&A]
Hello, 
my service just collapses at some point. Then only a systemctl restart UDM-iptv helps. Where can I find the UDM-iptv.service? I would like to increase the time to 10 or 15 sec. Or what can I do?

Please share the following output with the developers:
=== Configuration ===
WAN Interface: ppp0
WAN VLAN: 0 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 224.0.0.0/4 232.0.0.0/16 87.141.0.0/16
LAN Interfaces: br0 br40 br50
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
57: ppp0: <POINTOPOINT,MULTICAST,NOARP,UP,LOWER_UP> mtu 1492 qdisc pfifo_fast state UNKNOWN group default qlen 3
    inet 79.252.246.99 peer 62.155.210.40/32 scope global ppp0
       valid_lft forever preferred_lft forever
62.155.210.40 proto kernel scope link src 79.252.246.99

**Comment by fabianishere** [comment]
You can edit the service as follows:

```bash
systemctl edit udm-iptv.service
```

**Reply by jensus11** [reply]
![0342C8D1-D782-42B5-A4C7-7B9B9DEF3132](https://user-images.githubusercontent.com/17987533/191831065-76a03863-7bb5-4863-a932-2f777a34141c.jpeg)
No, that's not possible.  No file is found.  The file shown is empty.  That's why it doesn't work for me that the service is restarted.

**Reply by fabianishere** [reply]
What about:

```bash
vim /lib/systemd/system/udm-iptv.service
```

**Comment by jensus11** [comment]
Great, now I could edit the file.  What does restartsec=5s do?

**Reply by fabianishere** [reply]
https://www.freedesktop.org/software/systemd/man/systemd.service.html#RestartSec=
